### PR TITLE
Add troubleshoot doc on chectl

### PR DESCRIPTION
**Pull request by johnmcollier**
Signed-off-by: John Collier <John.J.Collier@ibm.com>

### CODE REVIEW

**Comment by johnmcollier** [comment]
@micgibso Please review

File: docs/_documentations/troubleshooting.md [hunk 1]
```diff
@@ -500,6 +500,33 @@ These steps reproduce the issue:
...
+OS: RHEL
+Che version: 7.5.1
+IDE extension version: 0.8.0
+IDE version: 7.5.1
+Kubernetes cluster: Red Hat OpenShift on IBM Cloud
+Action/Topic: 
+Issue type: bug/info
+Issue link: https://github.com/eclipse/codewind/issues/1806
+-->
+## Che sometimes fails to re-install successfully on Red Hat OpenShift on IBM Cloud
```
Comment (sishida) [inline docs/_documentations/troubleshooting.md]: "reinstall" doesn't need the hyphen.

File: docs/_documentations/troubleshooting.md [hunk 2]
```diff
@@ -500,6 +500,33 @@ These steps reproduce the issue:
...
+Sometimes when Che is re-installed on Red Hat OpenShift on IBM Cloud, the install may fail with the following error:
```
Comment (sishida) [inline docs/_documentations/troubleshooting.md]: Bold indicates what should be changed: "Sometimes when Che is **reinstalled** on Red Hat OpenShift on IBM Cloud, the **installation** **might** fail with the following error:"

File: docs/_documentations/troubleshooting.md [hunk 3]
```diff
@@ -500,6 +500,33 @@ These steps reproduce the issue:
...
+    ❯ PostgreSQL pod bootstrap
+      ✔ scheduling...done.
+      ✔ downloading images...done.
+      ✖ starting
+        → ERR_TIMEOUT: Timeout set to pod ready timeout 130000
+```
+
+This is due to permissions issues in the namespace.
+
+To resolve this, install Che into a namespace other than `che` with the `--chenamespace` flag when running `chectl server:start`.
```
Comment (sishida) [inline docs/_documentations/troubleshooting.md]: Recommended change: "To resolve **the error**,..."

**Comment by sishida** [comment]
@johnmcollier Hi, John, I knew you'd be offline, so I went ahead and copied your changes to a new PR and added my edits to the new PR: https://github.com/eclipse/codewind-docs/pull/314 The new PR is merged, so I am closing this one. Thank you! 😃

**Pull request closed without merging**